src/components/awards/Awards.tsx

```diff
@@ -1,99 +1,64 @@
-import GoodDA from "../../assets/goodDA.svg";
-import styles from "./awards.module.css";
+import styles from "./awards.module.scss";
 
 export default function Awards() {
   return (
-    <div className={styles.awards}>
-      <div className={styles.awards_headers}>
-        <h1 className={styles.awards_headers__header}>
+    <section className={styles.root}>
+      <div className={styles.headers}>
+        <h1 className={styles.header}>
           Make <i>your home</i> an ode to joy
         </h1>
-        <h2 className={styles.awards_headers__postheader}>
+        <p className={styles.subheader}>
           We turn your empty house to a lovely home, making the compact spaces
           with sapce saving furnitures. Making the unique tastes of yours into
           reality!
-        </h2>
+        </p>
       </div>
-      <div className={styles.awards_hero}>
-        <div className={styles.awards_hero__circle}>
-          <Arrow className="absolute top-11 right-11" alt="Arrow" />
-          <svg
-            className=" rotate-[107deg]"
-            viewBox="-15.5 75.5  130 130"
-            width="120"
-            height="120"
-          >
-            <defs>
-              <path
-                id="MyPath"
-                fill="none"
-                stroke="red"
-                d="M50,95 a 45,45 0 1,1 -0.1,0"
-              />
-            </defs>
+      <div className={styles.hero}>
+        <div className="container text-center lg:text-left">
+          <div className={styles.circle}>
+            <Arrow className="absolute right-10 top-10" alt="Arrow" />
+            <svg
+              className=" rotate-[107deg]"
+              viewBox="-15.5 75.5  130 130"
+              width="113"
+              height="113"
+            >
+              <defs>
+                <path
+                  id="MyPath"
+                  fill="none"
+                  stroke="red"
+                  d="M50,95 a 45,45 0 1,1 -0.1,0"
+                />
+              </defs>
 
-            <text fontSize="12" fontFamily="Outfit" fill="white">
-              <textPath className="tracking-[.27em]" href="#MyPath">
-                GET IN TOUCH • GET IN TOUCH •
-              </textPath>
-            </text>
-          </svg>
+              <text fontSize="12" fontFamily="Outfit" fill="white">
+                <textPath className="tracking-[.27em]" href="#MyPath">
+                  GET IN TOUCH • GET IN TOUCH •
+                </textPath>
+              </text>
+            </svg>
+          </div>
         </div>
-        <div className="relative bottom-16 overflow-hidden">
+        <div className="relative bottom-16 w-full overflow-hidden">
           <Image
-            className="h-screen object-cover "
+            className=" h-[640px] w-full object-cover"
             alt="Main Img"
             src={heroImg}
+            sizes="(max-width: 1200px) 100vw,
+              (max-width: 768px) 50vw,
+              20vw"
```

Comment:
I think this is not needed here cause you always have this image stretched to full screen width.
You will have your image blurred a lot on 1200px+ screens because you set 20vw here.